README.md

```diff
@@ -1,310 +1,320 @@
+
+
+
+### Build from Source {#source}
+
+The following instructions are for developers and contributors to cuGraph OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuGraph from source and contribute to its development.  Other operating systems may be compatible, but are not currently tested.
+
+The cuGraph package include both a C/C++ CUDA portion and a python portion.  Both libraries need to be installed in order for cuGraph to operate correctly.  
+
+The following instructions are tested on Linux systems.
+
+
+
+#### Prerequisites
+
+Compiler requirement:
+
+* `gcc`     version 5.4+
+* `nvcc`    version 9.2
+* `cmake`   version 3.12
+
+
+
+CUDA requirement:
+
+* CUDA 9.2+
+* NVIDIA driver 396.44+
+* Pascal architecture or better
+
+You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
+
+
+
+Since `cmake` will download and build Apache Arrow you may need to install Boost C++ (version 1.58+) before running
+`cmake`:
+
+```bash
+# Install Boost C++ for Ubuntu 16.04/18.04
+$ sudo apt-get install libboost-all-dev
+```
+
+or
+
+```bash
+# Install Boost C++ for Conda
+$ conda install -c conda-forge boost
+```
+
+
+
+#### Build and Install the C/C++ CUDA components
+
+To install cuGraph from source, ensure the dependencies are met and follow the steps below:
+
+1) Clone the repository and submodules
+
+  ```bash
+  # Set the localtion to cuGraph in an environment variable CUGRAPH_HOME 
+  export CUGRAPH_HOME=$(pwd)/cugraph
+
+  # Download the cuGraph repo
+  git clone https://github.com/rapidsai/cugraph.git $CUGRAPH_HOME
+
+  # Next load all the submodules
+  cd $CUGRAPH_HOME
+  git submodule update --init --remote --recursive
+  ```
+
+
+
+2) Create the conda development environment 
+
+​	A)   Building the `master` branch uses the `cugraph_dev` environment
+
+```bash
+# create the conda environment (assuming in base `cugraph` directory)
+# for CUDA 9.2
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev.yml
+
+# for CUDA 10
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.yml
+
+# activate the environment
+conda activate cugraph_dev 
+
+# to deactivate an environment
+conda deactivate
+```
+
+
+
+​	B) Create the conda development environment `cugraph_nightly`  
+
+If you are  on the latest development branch then you must use the `cugraph_nightly` environment.  The latest cuGraph code uses the latest cuDF features that might not yet be in the master branch.  To work off of the latest development branch, which could be unstable, use the nightly build environment.  
+
+```bash
+# create the conda environment (assuming in base `cugraph` directory)
+conda env create --name cugraph_nightly --file conda/environments/cugraph_nightly.yml

```

Comment:
We should probably add a coda 10 version of the cugraph_nightly.yml file.  Perhaps we should add an issue for this?  This could be deferred until 0.7, since the nightly builds won't be necessary to build with 0.6 once we release.

Reply:
Let's push it since it only impact new feature developers who can edit the file

Reply:
new yml file created